.github/workflows/build.yml

```
@@ -10,12 +10,11 @@ on:
 
 jobs:
   build:
-    runs-on: windows-2022
-
     strategy:
       matrix:
         node-version: [14.x, 16.x, 18.x, 20.x]
-
+        os: [windows-2022, ubuntu-2204]
```

```suggestion
        os: [windows-2022, ubuntu-latest]
```

Due to typo, runner stays stuck in endless waiting state.
use `ubuntu-22.04` or to prevent updating workflow files on each release by `ubuntu-latest`

---

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners